src/lib/SideBar.svelte

```diff
@@ -21,15 +20,48 @@
 		{ icon: 'fab fa-discord', info: 'Discord', activity: null },
 		{ icon: 'fab fa-github', info: 'GitHub', activity: null },
 	];
-
-	let activeInfo = null;
+	let activeInfo = null; // Tracks currently visible info
+	let lastHoveredInfo = null; // Tracks last hovered info
+	let hideTimeout = 0; // Timeout for hiding info panel
 
 	function showInfo(info) {
-		activeInfo = info;
+		// Clears and reset the active hideTimeout to stop the panel from being hidden
+		if (hideTimeout) {
+			clearTimeout(hideTimeout);
+			hideTimeout = 0;
+		}
+		// Updates activeInfo and lastHoveredInfo to track the hovered icon, then shows the panel with the hovered icon's info
+		if (info !== lastHoveredInfo) {
```

Comment:
I am not seeing the rationale for lastHoveredInfo here. Maybe at some point in the past we had some grace time before switching from one panel to another?